---
title: |  
  | Integrating macroecology 
  | into macroevolutionary models:
  | Evolution in Australian reptiles
author: "NSF Postdoc Outline"
date: \today
output:
  #word_document: default
  pdf_document: #default
      toc: true
      #number_sections: true
fontsize: 11pt
geometry: margin = 1in
indent: true
csl: /Users/Ian/Google.Drive/R.Analyses/rMarkdown_CSLs/proceedings-of-the-royal-society-b.csl
bibliography: /Users/Ian/Google.Drive/PostDoc_Applications/MacroEcoMacroEvo/NSF_References.bib
linestretch: 0.9 #0.85 # use to adjust line spacing
linkcolor: magenta 
#urlcolor: magenta
---

\     
\     


[*NSF Postdoctoral Research Fellowships in Biology Solicitation*](https://www.nsf.gov/pubs/2015/nsf15501/nsf15501.htm)

\pagebreak

# Project Summary
## Overview:
Ecosystems are defined by both the organisms inhabiting them, and the interactions among these organisms. Quantifying the influence of these interactions on the evolution of the participants should therefore be a priority in evolutionary biology, but remains difficult [@Drury2017a; @Weber2017]. Instead, common comparative methods account for phylogeny, but ignore how cohabiting lineages influence one another phenotypically and spatially. My research goal is to extend our existing toolkit to include methods which more accurately account for macroecology in macroevolution. This includes incorporating (**1**) inter-lineage interactions, (**2**) intra-lineage variation, and (**3**) historical biogeography, to better explain observed patterns in trait diversity. The proposed research will require expanding my understanding of the intersections of evolutionary and computational biology. I will be guided in my journey by two experts in these fields. My computational and methodological mentor **Dr. Brian O'Meara** (*University of Tennessee, Knoxville*) is a world expert in comparative methods, bringing together mathematical advances with empirical evolutionary studies. My curatorial and biological mentor **Dr. Mike Lee** (*South Australian Museum*) has been actively researching macroevolution and the complex origins of reptiles for over two decades, incorporating the often disparate fields of morphological and molecular evolution. Together, these sponsors (and their institutions) provide the opportunity to expand my theoretical understanding of evolutionary biology, and the computational and methodological abilities to apply it.

## Intellectual Merit
Organismal interactions provide an important selective force for evolution. Predator-prey, plant-pollinator, and host-parasite relationships form the basis for ecosystems, and drive the accumulation and distribution of diversity. For a while now, both community ecologists and comparative evolutionary biologists have sought to understand how this diversity is built on local and phylogenetic scales. However, they have approached these questions from very different angles. Community ecology has been largely agnostic of evolution along phylogenies, and evolutionary biologists have relied on ecology-na√Øve comparative methods. Now, we are on the edge of being able to appropriately incorporate community interactions into phylogenetic comparative methods of macroevolution. I aim to improve our ability to identify the influence of interlineage interactions on community assembly and trait evolution, and better estimate the strength of these interactions. I will develop models that account for intralineage trait variation and biogeographic histories, which will make phylogenetic comparative methods more biologically meaningful. My research will blend together the fields of macroecology and macroevolution, and in the process, bring together ecologists with evolutionary biologists, and museum curators with computer scientists.

## Broader Impacts
During this project, I will create a number of resources for both the macroecological/macroevolutionary and general science communities. This includes curating an open-source ecological database of Australian reptiles (morphology, dietary, distributional data), designing new methods for modelling trait evolution in a phylogenetic community framework, and communicating my results through scientific publications and popular science engagements.
I plan to use collections of the *South Australian* and *Western Australian Museums*, and will integrate collected data back into the **Atlas of Living Australia** database. Working in museums will also afford me the opportunity to enrich the experience of others by interacting with the public, and collaborating with domestic and international colleagues, including undergraduate and postgraduate students. I have also outlined a plans for a workshop on phylogenetic comparative methods (see appendix). Finally, I enjoy outreach through social media, so will continue to use my position in the community to broadcast results in an entertaining and digestible "PopSci" manner.

\pagebreak

# Project Description

## Introduction
Organismal phenotypes are the result of genetic *and* environmental forces, and the interaction between them. When explaining patterns of phenotypic evolution, we can account for some genetic influence by incorporating a phylogeny and drift, recognizing that phenotypes tend to be more similar between closely related species. Descent with modification thus provides us with some expectations as to how phenotypes should evolve. But how do we account for influence of extrinsic biotic factors on observed traits? In response to these questions, and in an attempt to explain the accumulation of organismal diversity, the field of phylogenetic comparative methods was developed. Unfortunately, nearly all commonly used probabilistic models for describing phenotypic evolution are ecologically neutral, meaning they fail to account for species interactions with one another, and their environment. This results in lifeless models that mathematically describe the *pattern* of phenotypic evolution, but lack a biological explanation of the *process*. **My overarching research goal is to extend our existing toolkit to include a more diverse methodoligical framework which accounts for macroecology in macroevolution.** This includes incorporating inter-lineage interactions, intra-lineage variation, and historical biogeography, to better explain observed patterns in trait diversity. To develop these methods, I intend to use Australian reptiles as an empirical example.  
\    

**But, why Australian reptiles?** As an island continent, Australia is a natural laboratory for studies of evolution. With more than 1,000 species of squamate reptiles (lizards and snakes), extensive museum collections, and curated occurrence records, Australia presents a prime opportunity for investigating macroecological models of phenotypic evolution. Since parting ways with Antarctica, the Australian continent has been isolated for around 35 million years. This means that with the exception of a single preexisting Gondwanan clade (Pygopodoid geckos), all other Australian reptile groups have arrived from Asia and radiated in situ. This includes iconic radiations of pythons, monitor lizards, elapid snakes, skinks, agamid lizards, and blind snakes. Australian institutions also provide extensive collections (**500,000+**), and carefully curated occurrence records, making it a prime location for investigating macroecological models of phenotypic evolution.

## Research Plan

### Research Objectives
The overarching goal of this project is to better account for macroecological processes in macroevolutionary methods. This includes designing new tools *and* implementing them to tackle empirical evolutionary questions. We can classify the broad aims of this study into four specific questions: 

**1.**  Does including interspecific interactions improve our inferences of phenotypic evolution?  

**2.**  How do interspecific interactions affect the spatial evolution and distribution of species?  

**3.**  Can we build an extensive ecological dataset of Australian reptiles on a continental scale?

**4.**  Is competitive signal identifiable from phenotypic and spatial data of Australian reptiles?

### Methods
This research depends directly on the development of novel methods and the collection and curation of a specimen-level eco-morphological database (of Australian reptiles). The ultimate goal is to integrate these to directly address our questions of interlineage interactions using empirical data. 

I will use Australian museum specimens to collect existing and new data on reptiles to develop an ecological database that will be incorporated into the Atlas of Living Australia (ALA) [@ALA]. The approaches used to collect and integrate the data will likely provide a benchmark for future ALA projects across other Australian organismal groups. The process of developing the comparative methods will also be of direct interest for those hoping to follow a similar methodology.

I anticipate that the incorporation of multiple factors influencing trait and spatial evolution may require complex, parameter-rich models. Methods for modelling univariate continuous traits have traditionally depended on the ability to directly estimate the likelihood of the data given the model. With increasing complexity, likelihood calculations become difficult or intractable, and so I suggest the development instead, of an **Approximate Bayesian Computation** (ABC) framework. ABC methods rely on the ability to simulate data under a designated model, then determine the similarity between simulated and observed summary statistics. In this way, ABC is a remarkably flexible and valuable tool for complex model-fitting, and will be a boon to this proposal. 

#### Objective 1: Does including interspecific interactions improve our inferences of phenotypic evolution?
Interspecific interactions act as a strong selective force on organismal phenotypes. Interactions may be competitive (vying for resources), mutualistic (beneficial to both parties), commensal, parasitic, or predatory, and dictate a variety of dynamic responses. This has led to theories of character displacement, competitive exclusion, and phenotypic matching, among others. In a classic anecdote, Darwin and Wallace each considered a Madagascan orchid with a foot long nectary. They then independently determined there must be a moth pollinator with an equally long proboscis to match. The later discovery of just such a moth was ultimately less of a surprise than it was a fulfillment of the expectation that organismal interactions can dictate phenotypic evolution, even to absurd extremes. 

Despite us witnessing these interactions every day, probabilistic evolutionary models still treat *all* lineages identically, and until just recently [@Drury2018; @Drury2017a; @Drury2017b; @Manceau2016], ignored interactions altogether. Building off conceptual work by Nuismer & Harmon [@Nuismer2015], Drury et al. [@Drury2017b] and Manceau et al. [@Manceau2016] elegantly integrated a system of ordinary differential equations for estimating the effect of competition on trait evolution in a maximum likelihood framework. We can thus estimate a parameter *S* which describes the strength of the interaction, as well as the polarity: negative values of *S* indicate competition or repulsion, positive values indicate attraction towards common values. In in its most simplistic form, the *S* parameter interacts with the mean trait values of the interacting lineages (vector *X*~t~), to reflect their relationship. To take into account changes through evolutionary time, the *S* parameter further interacts with the evolutionary rate ($\sigma$), and drift (*d*), to dictate the trajectory of trait evolution. As part of my PhD research, I have extended this method to account for historical biogeography (inferred separately), which attempts to capture the reality that not all lineages occur sympatrically (an assumption of the original model) (Figure 1). However, one could imagine that a number of factors dictate the interactions between lineages. Niches can be partitioned by ecological means (diel activity, microhabitat, etc.), and I think these aspects could be incorporated by including a number of interaction matrices, similar to how geography currently is (Figure 1). Alternatively, it is unlikely that all lineages are interacting with identical intensity, i.e. estimating a single *S* parameter. Instead, lineages may interact differently based upon the extent of their geographic (broad or subset sympatry) or active period (nocturnal, crepuscular, diurnal) overlap, or even some uncaptured aspect of their ecology (hidden states). The estimation of multiple interaction (*S*), rate ($\sigma$), and drift (*d*) parameters however, becomes increasingly difficult in a **maximum likelihood** framework, encouraging the development for methods which do not require full-likelihood calculations, such as **ABC** [@Beaumont2010; @Csillery2010]. 


#### Objective 2: How do interspecific interactions affect the spatial evolution and distribution of species? 
The distribution of species across a landscape can be dictated by both intrinsic and extrinsic influences. In maybe the most obvious examples, habitats may "filter" for species with similar ecologies, or instead, competition might result in exclusion from a given ecosystem. The strength of these interactions (be it competitive or mutualistic) therefore govern the extent to which species cohabitate. This suggests an intersection between geographic distributions and phenotypes. Unfortunately, no current biogeographic models incorporate trait values or competition in their reconstructions [@Sukumaran2018]. Of perhaps greater concern is that in nearly all biogeographic models, species distributions are simplified either to the centroid of their range, or discretized into island, continental, or biome bins. What is lost in translation is the biology of the organism: species ranges may be irregularly shaped, of inconsistent density, and/or vary greatly in size [@Quintero2015; @Lemmon2008]. This may be a result of a number of influences such as available habitat and physiological limits, but is likely also dictated by interactions with related (and unrelated) species. I suggest the development of a spatially explicit forward-time simulation-based framework to approximate the processes resulting in current distribution patterns. This framework will incorporate ecology and species traits to inform how interactions affect the spatial evolution of species. 

```{r pressure, echo=FALSE, out.width = '90%', fig.cap="A partial schematic of the process for incorporating biogeographic histories and competition into trait evolution. Left to right: a phylogeny, the spatial distribution of extant taxa, and an interaction matrix for calculating the influence of lineages on one another. Note, it is a symmetric matrix (1 sympatric, 0 allopatric), which allows only sympatric taxa to interact.", eval=T}
# change 'eval=FALSE' to 'eval=TRUE' to include this plot
knitr::include_graphics("/Users/Ian/Google.Drive/PostDoc_Applications/RASE_diagram.png")
# for some reason this appears to be the preferred way to include a plot in the text
```


#### Objective 3: Can we build an extensive ecological dataset of Australian reptiles on a continental scale?
Existing eco-morphological databases cover much of the Australian vertebrate fauna, including mammals [@Pantheria] and birds [@Garnett2015; @EltonTraits], but reptile taxa remain poorly represented [@Meiri2018]. This is surprising given the number of studies of Australian reptile phylogenetics, ecology, and morphology, and remarkable museum collections. In association with the **Atlas of Living Australia** (ALA) [@ALA], I intend to collate existing, undatabased ecological and morphological information with newly collected data, to span the entirety of Australian reptile taxa (>1000 species). The ALA provides a portal for interacting with museum and citizen-scientist records of Australian organismal diversity, comprising 78,492,847 records of 123,364 species. Native reptiles account for over 1 million of these records, which include important spatial, temporal, and taxonomic information about these taxa. Noticeably absent however, are phenotypic data for any of the 500,000+ reptiles in museum collections. In a departure from existing databasing methods, I hope to integrate morphological and ecological data, along with metadata for all viewed specimens back into the ALA database. My hope is that instead of summarizing this information to species means, I can contribute a database which is more flexible to taxonomic changes and better describes intraspecific variation [@Troudet2018]. As a current research associate of the Western Australian Museum (130,000+ reptile records) and a proposed associate of the South Australian Museum (55,000+ reptile records), I would have access to sampling for the entirety of the continent, making this type of data collection feasible. A resource of this kind would be a world-first, providing invaluable information to future researchers, and returning inalienable data to museums about their own collections.   

#### Objective 4: Is competitive signal identifiable from phenotypic and spatial data of Australian reptiles?
This question represents the synthesis of the three previous objectives. Broadly, my intention is to develop methods that may be of interest to other researchers studying macroevolution and macroecology. However, it is also important to ask these questions empirically, **by employing existing biological collections**. Australian reptiles are particularly well suited for these objectives because they represent a number of species rich, eco-morphologically diverse, endemic radiations. Diversification *in-situ* reduces the influence of immigration and emigration, and simplifies our research. Building on phylogenomic research from my PhD and others, we now have strong species-level time-calibrated phylogenetic hypotheses for nearly all continental reptile groups. This makes macroevolutionary inferences tractable. The Atlas of Living Australia also provides an unparalleled resource for species distributions based on museum occurrence records, for all described taxa. Given existing data and resources, and the access to material, I *honestly* believe that this is one of the best possible systems to focus this research on. 

### Significance and Broader Impacts
Since Joe Felsenstein [@Felsenstein1985] highlighted the nonindependence of species in comparative methods (owing to their underlying relatedness---phylogeny), Phylogenetic Comparative Methods (PCMs) have become commonplace throughout evolutionary and ecological research. This is likely the result of expanding phylogenetic and computational resources, and user-friendly implementations [@Paradis2004; @Revell2011]. Nonetheless, comparative methods have exploded as a popular area of study and application, particularly in the last ten years. In their most basic form, PCMs can provide information about-- and correct for phylogenetic signal present in data. Their flexible design however, means that they can be easily extended to incorporate diverse data types. Regardless, many of the inferences we can make from PCMs are intuitive, which makes them exciting and digestible for popular science media. 

My proposed research builds on existing ecological theory and macroevolutionary methods, synthesizing the two. I anticipate that the output of this project will be valuable both empirically and methodologically. The construction of an ecomorphological database will provide a resource for a diverse array of biologists working from population to macroevolutionary scales. The comparative ABC method design will also make available a generalizable framework for creating evolutionary models. I aim to publish the data, methods, and empirical findings, and look forward to sharing our results with colleagues and the general public. 

## Training Objectives and Career Development
Evolutionary biology is increasingly intersecting with computational biology. To expand my versatility in phylogenetic comparative methods, I aim to first extend my abilities in statistical modelling. This means strengthening the connection between theoretical understanding and computational implementation of macroevolutionary processes. I have carefully chosen two sponsors who are experts in these fields, and in bridging the divide between them, bringing together the conceptual and empirical. 

As a self-taught coder, there remain a number of shortcomings in my computational abilities, particularly with regards to efficiency. Dr. O'Meara is an experienced member of the new-wave of code-conscious evolutionary biologists, and is the author of popular comparative methods packages treePL, brownie, and OUwie [@OUwie2012; @Brownie2009]. I look forward to also developing a better understanding of the underlying mathematics behind current methods. Dr. Lee is a world-expert in Bayesian modelling, and I will receive training from him in directly in Bayesian computation. His experience in functional morphology will also be of benefit to my interest in lineage interactions. 

I have always been interested in a career that merges the creativity of research with the outreach of science communication. I have been and will remain a strong activist for science education and literacy, and am committed to fulfilling this dream. I believe that the training provided by Drs. O'Meara and Lee will help to make me a rising star in the field of evolutionary biology, however this is not my exclusive goal. The South Australian Museum, and proposed workshops, will also provide the opportunity to continue my development as an educator and science communicator. I feel that as a scientist, and a publicly funded researcher, I owe a debt to make my research digestible and accessible to the public. 

## Sponsoring Scientist and Institution Justificiation

I have chosen Drs. Lee and O'Meara specifically for their skill sets, research interests, and proven track records of pushing the evolutionary biology community forward. They are both creative and judicious thinkers, who I admire for their contributions to science and broader education.  

**Dr. Brian O'Meara** (*Methods/Biology*) is an Associate Professor in the Department of Ecology & Evolutionary Biology at the University of Tennessee, Knoxville. His lab group work to build and apply comparative methods in phylogenetics. He is known for his expertise in comparative methods and phylogenetics, and his community-mindedness. Dr. O'Meara is a leader in the new-guard of biology, both a skilled researcher, and conscientious advocate. 

**Dr. Michael Lee** (*Theory/Biology*) is the Leader of Research in Palaeontology at the South Australian Museum, and holds a joint appointment in the Department of Biological Sciences at Flinders University of South Australia. Dr. Lee is primarily an evolutionary biologist, who works at the interface of palaeontology, neontology, and computational biology. 

I will spend the first year (Y1) of this fellowship at the **South Australian Museum** (SAM) in Adelaide. The SAM is an amazing resource for researchers across the natural sciences. Though I will be primarily mentored by Dr. Lee during this period, I will also be working remotely with Dr. O'Meara, and building relationships with SAM colleagues. I expect that sometime after the first year, I will transition my position from the data collection phase in Australia, to the methods development phase with Dr. O'Meara in the USA. Both institutions are world-recognized, and have state of the art facilities. 


## Timetable
The requested duration of this fellowship is **3 years**. The current timetable assumes Year 1 and part of Year 2 will be spent at the South Australian Museum in Adelaide, Australia. The remainder of Year 2 and Year 3 will be spent at the University of Tennessee---Knoxville, USA. 
```{r echo = FALSE}
library(knitr); library(pander)

df <- data.frame(Year = c("Y1","Y2", "Y3"),
                 Research = c("Collect morphological data. Build reptile database, and integrate it into ALA[^1]",
                              "Build ABC^1^ framework for PCMs^1^. Implement empirically to reptiles",
                              "Package and release ABC framework for lineage interactions"),
                 Training = c("Learn databasing methods. Work with the ALA", "Build software development skills", "Extend development skills. Train others on methods"),
                 Dissemination = c("Release extended specimen data format for ALA, and present database",
                                  "Present methods and results at scientific conferences. Publish project   pre-prints",
                                  "Publish methods and empirical findings"))
pander(df, split.cell=20)
```

[^1]:ALA--Atlas of Living Australia; ABC--Approximate Bayesian Computation; PCM--Phylogenetic Comparative Model

\pagebreak

# Biographic Sketch (2pg CV)
*I think I have this bit already covered*

\pagebreak

# Current and Pending Support
The only current application for funding is through the University of California, Davis Center for Population Biology. This is a two-year self-directed fellowship, to be started sometime September--December 2019. The advertised salary is $50,000 per annum, with $5,000 of research support. The application was submitted on November 1, 2018, and is awaiting review. 

\  

I have no other currently submitted, or planned applications for support. 

\pagebreak

# Supplementary Documentation
*I've already submitted my workshop proposal.*

\pagebreak

## Dissertation Research Abstract

### Macroevolution in a changing Australia

Changes to the global climate can promote macroevolutionary and macroecological turnover by either abiotic or biotic drivers, or both. Climatic changes may proceed over long or short time periods, varying in intensity from mild to extreme, and as a result, changes to macroevolutionary patterns may respond in kind. It remains unknown however, how *predictable* organismal responses may be, particularly across ecologically diverse groups. My dissertation research has largely focused on how changing environments have impacted the accumulation of diversity of Australian vertebrates since the continent's isolation ~35 million years ago. 

\  

The Miocene epoch (23‚Äì5.3 Ma) has reportedly figured prominently in the diversification of many extant faunal groups. This is likely the result of climatic instability, fluctuating atmospheric CO~2~ concentrations and floral biome turnover. Across the globe, Late Miocene cooling coincided with the birth and expansion of arid biomes and contraction of more mesic ones, as C3 forest and woodland plants were replaced by C4 savannah and grasslands. In Australia, this resulted in a dramatic restructuring of habitats, transitioning the continent from warm wet forests, to the barren red sands of today's "Outback". To investigate this idea, we can search for signature of these climatic shifts in the evolutionary history of Australia's animals. 

\  

To test for signal from empirical data, I employed phylogenetic comparative methods to link diversification patterns with climate-dictated processes. Given a set of taxa and tissue samples, we produced genetic data via traditional Sanger-sequencing and exon-capture methods. This allowed us to infer relationships among the sampled taxa, providing a phylogenetic framework. Using these phylogenies, we investigated (**1**) rates and trends in speciation and extinction, (**2**) biogeographic histories including trends in the prevailing geographic mode of speciation, and (**3**) the evolution of organismal traits. Our findings suggest that heterogeneity in the diversification of Australian animals has been largely dictated by biome restructuring through the Miocene and Plio-Pleistocene. This is evident by (**1**) increased speciation in arid and dispersal back into mesic habitats, (**2**) increased allopatry as a result of fracturing mesic habitats, and (**3**) decreasing rates of phenotypic evolution due to niche conservatism. We provide evidence that broad abiotic and biotic influences can result in similar patterns across cohabiting groups with disparate ecologies. While Australia is unique in its forms of diversity, its biogeographic and phenotypic patterns have probably been shaped by the same processes occurring elsewhere. 

\pagebreak

## Sponsoring Scientists Statement
Drs. Brian O'Meara and Mike Lee are highly supportive of Ian Brennan's postdoctoral fellowship application. The proposed research will provide a logical extension of existing phylogenetic comparative methods, incorporating ecological interactions into models of macroevolution. This project will help Ian build his own independent research program, and learn the skills to become a successful biological software developer. His design for developing resources for integrating existing morphological data into the Atlas of Living Australia is a novel and praiseworthy endeavor. He will be supported along his journey by two world-class institutions.  

### Current Research and Funding
Dr. O‚ÄôMeara and his lab group build and apply comparative methods in phylogenetics. Ongoing works includes a model linking population genetics and phylogenetics models, approximate Bayesian computational approaches for traits evolving on phylogenies, methods to calibrate phylogenies and make these available to the public, outreach approaches letting educators use the tree of life, empirical work on dung beetle phylogeny and fish evolution, reviews of biogeography and diversification methods, and continued extension of diversification models. Current funding includes NSF DBI-1458603 (open infrastructure for phylogenetic knowledge), NSF DEB-1453424 (CAREER grant on ABC for comparative methods), NSF DEB-1355033 (population genetics and phylogenetics), as well as a DDIG on fish evolution for the work of a graduate student in the lab. There is also a grant in review on innovations in graduate education and two proposals in preparation; one on Rules of Life regarding evolutionary layers from hidden state models, and one on diversification models with continuous, evolving rates. The fellow would work directly with Dr. O‚ÄôMeara as well as participate in the vibrant lab community. The research most closely complements the ABC trait evolution work. In this work, repulsion between species is possible, unlike many comparative methods, but it is extremely simple, being based on one trait. The current proposal has a much richer, more biologically and geographically realistic model for interactions between species, and so the two approaches can help one another. We also work on databases and interfaces to databases within the lab as well as web scraping for data in general, so assisting in the analysis of ALA data is squarely within our skill set.

Prof Lee is an evolutionary biologist whose work integrates genetic and phenotypic data to bridge the extinct/extant divide. This requires extensive experience with empirical morphological data, including an understanding of the underlying ecology and functional morphology, paired with finesse with molecular genetic and genomic analyses. He implements much of his research computationally in a Bayesian framework, and is considered one of the leading global experts in Bayesian phylogenetic analysis of evolutionary data. Though he has worked across numerous phyla, much of his research is on reptiles, focusing on their evolution and biogeography, and is particularly drawn to Australian lizards and snakes. His work is currently supported by Australian Research Council (ARC) Discovery and Linkage grants, that aim to (1) understand the evolutionary problems and ecological settings of the fish-tetrapod transition (2016-18), (2) integrate fossils and genomes to resolve the early evolution of snakes (2016-18, (3) investigate phenotypic and venom evolution in Australian tiger snakes (2017-19). There is no direct financial overlap for Ian‚Äôs proposed research, though there is shared interest in the empirical system, Australian reptiles. Ian‚Äôs project would be a welcomed contribution to the understanding of Australia‚Äôs rich reptile diversity.  


### Developmental Plan
In his first year, Ian will join the South Australian Museum as a Research Associate, under the mentorship of Prof. Lee. The Lee lab focuses on resolving paleontological/neontological evolutionary questions using diverse sets of data and cutting-edge Bayesian methods. There are a number of PhD student, associates, and faculty researchers employed or affiliated with the SAM (including Prof. Steve Donnellan, Dr. Mark Hutchinson; see below), who form a high-calibre group for evolutionary and methodological discussions. The SAM also provides a great opportunity for community-outreach and public science discussions and awareness. We expect Ian will contribute to existing programs where interested. 

Working remotely, Ian will also be an active member of Dr. O'Meara's lab group in Tennessee. Dr. O‚ÄôMeara has trained sixteen postdocs directly and, as associate director for postdoctoral mentoring for the National Institute for Biological and Mathematical Synthesis, assisted in the vocational training for many more. Training starts with a frank conversation with the postdoc on their professional goals and their strengths and weaknesses. This translates into a plan for acquiring necessary skills: recommendations for reading and exercises and discussions throughout the training. For this project, skills to work on will be developing and testing comparative methods and software engineering best practices, among others. After completing his time at the SAM, Ian will transition fully to a position in the O'Meara lab for the remainder of the fellowship. 

### Sponsorship Roles and Resources
The proposed research requires Ian to develop an efficient plan for data collection and management, learn new computational skills, and delve deeply into macroevolutionary theory, to unite computational and evolutionary biology. To reach this goal, we intend to support him in all pursuits. We will meet regularly, but informally, to discuss conceptual and empirical progress and stimulate ideas. In Dr. O'Meara's lab, every member meets individually weekly. We also have a weekly lab meeting that is done via teleconference for all participants, so that all participate equally regardless of location: the lab frequently has members who do all or some of their postdoctoral work away from Tennessee (members have worked from France, Texas, West Virginia, and Colorado). There is also an active lab chat room and occasional mini hackathons to help build teamwork and computational skills as well as online discussion groups for papers. Looking over paper drafts and job application materials is standard, as are frequent discussion on current topics in evolution, best practices in coding, and vocational training (how to teach, how to write research statements and grants, and similar). The lab has access to computers to use for any computationally intensive analyses or simulations and servers to use to host any tools.

It is important to respect the independence of early career researchers, and so we pledge to provide the tools necessary for success. The SAM is full of some of Australia‚Äôs best and brightest in ecology, natural history, and of course herpetology. This includes Dr. Mark Hutchinson (Senior Research Scientist‚Äîan expert in Australian Herpetology), Dr. Steve Donnellan (Chief Research Scientist‚Äîvertebrate evolution and biogeography), who are existing colleagues of Ian.  Dr. Kate Sanders (molecular ecology of reptiles) and Dr. Emma Sherratt (geometric morphometrics) also have joint affiliations with the SA museum and neighbouring Adelaide University.  The museum also houses the oldest and largest collection of DNA samples in the Southern Hemisphere, a cutting-edge next-gen DNA laboratory (led by Donnellan), and unparalleled alcohol and skeletal specimen holdings of Australian reptiles.


### Limitations
Upon completion of this postdoctoral fellowship, the collected data will be integrated into a freely available database, and the methods distributed freely as well. This position will help develop Ian as a rising star in the field, and we encourage him to pursue further research without any restrictions or limitations. He may continue to collaborate or not as he sees fit.




\pagebreak

## Documentation from Collection Director(s)

\pagebreak

```{r, echo=FALSE, out.width = '105%', eval=T}
# change 'eval=FALSE' to 'eval=TRUE' to include this plot
knitr::include_graphics("/Users/Ian/Google.Drive/PostDoc_Applications/SAM_header.png")
# for some reason this appears to be the preferred way to include a plot in the text
```

\  

\  

\  

\  

Dear Selection Panel,

\  

\  

\  

I am writing to provide evidence that Ian Brennan's proposed research would be permissible under the research and collections use policies of the South Australian Museum. We look forward to having him join our community of researchers. 
\  

\  

Additionally, Ian has been appointed a Research Associate of the Western Australian Museum, and so will have access to their facilities and resources as well.

\  

\  

\  

Sincerely,  
Michael S.Y. Lee  
Research Leader in Palaeontology  
South Australian Museum  

\pagebreak

## Data Management Plan
### Research Products
The goals of this project are to (**1**) create a generalizable probabilistic framework for incorporating interlineage interactions into models of trait and spatial evolution, and (**2**) collate and distribute an ecomorphological database for Australian reptiles. Both of these products will be freely available by the end of this fellowship.

### Data Format and Content
All progress with data collection and methods development will be diaried in a version-controlled repository on Github [www.github.com/iangbrennan](github.com/iangbrennan). Code in development will be available to collaborators, and will primarily be written in the opensource R and C++ programming languages. They will be accompanied by Rmarkdown tutorials, to facilitate their use. 

\  

Raw data for the reptile trait database will be stored as .csv files, before being incorporated into the Atlas of Living Australia. At the end of Year 1, the data will be made available in the ALA web portal, for use by all. 

### Data Access, Archiving, and Redistribution
The release of the reptile database and incorporation into ALA will be documented in a data specific journal such as *Nature: Scientific Data*, and/or as a note in a popular systematics or ecology journal such as *Ecology* or *Systematic Biology*. Moving forward, the data will be maintained by the ALA, though I will retain a personal backup on my Github page. Data licensing will be transferred to the ALA, and released to the public domain for broad use.

### Software Distribution
The comparative methods developed for this fellowship will be bundled into an R package, and distributed freely either via the Comprehensive R Archiving Network (CRAN), and backed up on Github. During development it will be stored in a version-controlled repository. 

### Roles and Responsibilities
Ian Brennan will be responsible for the collection, management, and archiving of all data. Dr. O'Meara will provide guidance on database development and management. The Atlas of Living Australia will provide help with data formatting and integration into the ALA system. 

\pagebreak

# 2 Letters of Reference (using Fastlane)

\pagebreak

# References  
