---
title: |  
  | Integrating macroecology 
  | into macroevolutionary models:
  | Evolution in Australian reptiles
author: "NSF Postdoc Outline"
date: \today
output:
  #word_document: default
  pdf_document: #default
      toc: true
      #number_sections: true
fontsize: 11pt
geometry: margin = 1in
indent: true
csl: /Users/Ian/Google.Drive/R.Analyses/rMarkdown_CSLs/proceedings-of-the-royal-society-b.csl
bibliography: /Users/Ian/Google.Drive/PostDoc_Applications/MacroEcoMacroEvo/NSF_References.bib
linestretch: 1 #0.85 # use to adjust line spacing
linkcolor: magenta 
#urlcolor: magenta
---

\     
\     


[*NSF Postdoctoral Research Fellowships in Biology Solicitation*](https://www.nsf.gov/pubs/2015/nsf15501/nsf15501.htm)

\pagebreak

# Project Summary (1pg max)
## Overview (include sponsoring scientists and institutions):
Ecosystems are defined by both the organisms inhabiting them, and the interactions among these organisms. Quantifying the influence of these interactions on the evolution of the participants should therefore be a priority in evolutionary biology, but remains difficult [@Drury2017a; @Weber2017]. Instead, common comparative methods account for phylogeny, but ignore how cohabiting lineages influence one another phenotypically and spatially. My research goal is to extend our existing toolkit to include methods which more accurately account for macroecology in macroevolution. This includes incorporating (**1**) inter-lineage interactions, (**2**) intra-lineage variation, and (**3**) historical biogeography, to better explain observed patterns in trait diversity.

## Intellectual Merit
Organismal interactions provide an important selective force for evolution. Predator-prey, plant-pollinator, and host-parasite relationships form the basis for ecosystems, and drive the accumulation and distribution of diversity. For a while now, both community ecologists and comparative evolutionary biologists have sought to understand how this diversity is built on local and phylogenetic scales. However, they have approached these questions from very different angles. Community ecology has been largely agnostic of evolution along phylogenies, and evolutionary biologists have relied on ecology-naïve comparative methods. Now, we are on the edge of being able to appropriately incorporate community interactions into phylogenetic comparative methods of macroevolution. My specific aims are to improve our ability to identify the influence of interlineage interactions on community assembly and trait evolution, and better estimate the strength of these interactions. By accounting for intralineage trait variation and biogeographic histories, we can make these methods more biologically meaningful. This research intends to blend together the fields of macroecology and macroevolution, and in the process, bring together ecologists with evolutionary biologists, and museum curators with computer scientists.

## Broader Impacts
For this project, I will create a number of resources for both the macroecological/macroevolutionary and general science communities. This includes curating an open-source ecological database of Australian reptiles (morphology, dietary, distributional data), designing new methods for modelling trait evolution in a phylogenetic community framework, and communicating my results through scientific publications and popular science engagements. My computational and methodological mentor Dr. Brian O'Meara (*University of Tennessee, Knoxville*) is a world expert in comparative methods, bringing together mathematical advances with empirical evolutionary studies. My curatorial and biological mentor Dr. Mike Lee (*South Australian Museum*) has been actively researching macroevolution and the complex origins of reptiles for over two decades, incorporating the often disparate fields of morphological and molecular evolution. Together, these sponsors (and their institutions) provide the opportunity to expand my theoretical understanding of evolutionary biology, and the computational and methodological abilities to apply it. In turn, I aim to enrich the experience of others by collaborating with domestic and international colleagues, including undergraduate and postgraduate students. I have also outlined a plans for a workshop on phylogenetic comparative methods (see appendix). Finally, I enjoy outreach through social media, so will continue to use my position in the community to broadcast results in an entertaining and digestible "PopSci" manner.
*workshop development*

\pagebreak

# Project Description (6pg max)

## Introduction
Organismal phenotypes are the result of genetic *and* environmental forces, and the interaction between them. When explaining patterns of phenotypic evolution, we can account for some genetic influence by incorporating a phylogeny and drift, recognizing that phenotypes tend to be more similar between closely related species. Descent with modification thus provides us with some expectations as to how phenotypes should evolve. But how do we account for influence of extrinsic biotic factors on observed traits? In response to these questions, and in an attempt to explain the accumulation of organismal diversity, the field of phylogenetic comparative methods was developed. Unfortunately, nearly all commonly used probabilistic models for describing phenotypic evolution are ecologically neutral, meaning they fail to account for species interactions with one another, and their environment. This results in lifeless models that mathematically describe the *pattern* of phenotypic evolution, but lack a biological explanation of the *process*. **My overarching research goal is to extend our existing toolkit to include a more diverse methodoligical framework which accounts for macroecology in macroevolution.** This includes incorporating inter-lineage interactions, intra-lineage variation, and historical biogeography, to better explain observed patterns in trait diversity. To develop these methods, I intend to use Australian reptiles as an empirical example.  
\    

**But, why Australian reptiles?** As an island continent, Australia is a natural laboratory for studies of evolution. With more than 1,000 species of squamate reptiles (lizards and snakes), extensive museum collections, and curated occurrence records, Australia presents a prime opportunity for investigating macroecological models of phenotypic evolution. Since parting ways with Antarctica, the Australian continent has been isolated for around 35 million years. This means that with exception a single preexisting Gondwanan clade (Pygopodoid geckos), all other Australian reptile groups have arrived from Asia and radiated in situ. This includes iconic radiations of pythons, monitor lizards, elapid snakes, skinks, agamid lizards, and blind snakes...

## Research Plan

### Research Objectives
The overarching goal of this project is to better account for macroecological processes in macroevolutionary methods. This includes designing new tools *and* implementing them to tackle empirical evolutionary questions. We can classify the broad aims of this study into four specific questions: 

*maybe I should fold question 4 into questions 1&2, to make this section more succinct?*

**1.**  Does including interspecific interactions improve our inferences of phenotypic evolution?  

**2.**  How do interspecific interactions affect the spatial evolution and distribution of species?  

**3.**  Can we build an extensive ecological dataset of Australian reptiles on a continental scale?

**4.**  Is competitive signal identifiable from phenotypic and spatial data of Australian reptiles?

### Methods
This research depends directly on the development of novel methods and the collection and curation of a species-level eco-morphological database (of Australian reptiles). The ultimate goal is to integrate these to directly address our questions of interlineage interactions using empirical data. We expect that the approaches used to collect the data and develop the comparative methods will also be of direct interest for those hoping to follow a similar methodology.

I anticipate that the incorporation of multiple factors influencing trait and spatial evolution may require complex, parameter-rich models. Methods for modelling univariate continuous traits have traditionally depended on the ability to directly estimate the likelihood of the data given the model. With increasing complexity, likelihood calculations become difficult or intractable, and so I suggest the development instead, of an Approximate Bayesian Computation (ABC) framework. (**Brian, maybe there's room to build on your TreEvo package?**). ABC methods rely on the ability to simulate data under a designated model, then determine the similarity between simulated and observed summary statistics. In this way, ABC is a remarkably flexible and valuable tool for complex model-fitting, and will be a boon to this proposal. 

#### Objective 1: Does including interspecific interactions improve our inferences of phenotypic evolution?
Interspecific interactions act as a strong selective force on organismal phenotypes. Interactions may be competitive (vying for resources), mutualistic (beneficial to both parties), commensal, parasitic, or predatory, and dictate a variety of dynamic responses. This has led to theories of character displacement, competitive exclusion, and phenotypic matching, among others. In a classic anecdote, Darwin and Wallace each considered a Madagascan orchid with a foot long nectary. They then independently determined there must be a moth pollinator with an equally long proboscis to match. The later discovery of just such a moth was ultimately less of a surprise than it was a fulfillment of the expectation that organismal interactions can dictate phenotypic evolution, even to absurd extremes. 

Despite witnessing these interactions every day, probabilistic evolutionary models still treat *all* lineages identically, and until just recently [@Drury2018; @Drury2017a; @Drury2017b; @Manceau2016], ignored interactions altogether. Building off conceptual work by Nuismer & Harmon (2015), Drury et al. (2016) and Manceau et al. (2016) elegantly integrated a system of ordinary differential equations for estimating the effect of competition on trait evolution in a maximum likelihood framework. We can thus estimate a parameter *S* which describes the strength of the interaction, as well as the polarity: negative values of *S* indicate competition or repulsion, positive values indicate attraction towards common values. The *S* parameter further interacts with the evolutionary rate ($\sigma$), and drift (*d*), to dictate the trajectory of trait evolution. As part of my PhD research, I have extended this method to account for historical biogeography (inferred separately), which attempts to capture the reality that not all lineages occur sympatrically (an assumption of the original model). However, one could imagine that a number of factors dictate the interactions between lineages. Niches can be partitioned by ecological means (diel activity, microhabitat, etc.), and I think these aspects could be incorporated by including a number of interaction matrices. Alternatively, it is unlikely that all lineages are interacting with identical intensity, i.e. estimating a single *S* parameter. Instead, lineages may interact differently based upon the extent of their geographic (broad or subset sympatry) or active period (nocturnal, crepuscular, diurnal) overlap, or even some uncaptured aspect of their ecology (hidden states). The estimation of multiple interaction (*S*), rate ($\sigma$), and drift (*d*) parameters however, becomes increasingly difficult in a **maximum likelihood** framework, encouraging the development for methods which do not require full-likelihood calculations, such as ABC (Beaumont, Csilléry). 

*I can probably cut the Darwin/Wallace anecdote*  


#### Objective 2: How do interspecific interactions affect the spatial evolution and distribution of species? 
The distribution of species across a landscape can be dictated by both intrinsic and extrinsic influences. In maybe the most obvious examples, habitats may "filter" for species with similar ecologies, or instead, competition might result in exclusion from a given ecosystem. 

#### Objective 3: Can we build an extensive ecological dataset of Australian reptiles on a continental scale?
Existing eco-morphological databases cover much of the Australian vertebrate fauna, including mammals (cite Pantheria) and birds (Garnett et al., 2014; EltonTraits), but reptile taxa remain poorly represented (Meiri, 2018). This is surprising given the number of studies of Australian reptile phylogenetics, ecology, and morphology, and remarkable museum collections. In association with the **OpenTraits** initiative, I intend to collate existing, undatabased ecological and morphological information with newly collected data, to span the entirety of Australian reptile taxa (>1000 species). In a departure from traditional databasing methods, I hope to include measurements, observations, and metadata for all viewed specimens. My hope is that instead of summarizing this information to species means, I can contribute a database which is more flexible to taxonomic changes and better describes intraspecific variation (Troudet etal.2018). As a current research associate of the Western Australian Museum (130,000+ reptile records) and a proposed associate of the South Australian Museum (55,000+ reptile records), I would have access to sampling for the entirety of the continent, making this type of data collection feasible. A resource of this kind would be a world-first, providing invaluable information to future researchers, and returning inalienable data to museums about their own collections.   

#### Objective 4: Is competitive signal identifiable from phenotypic and spatial data of Australian reptiles?
This question represents the synthesis of the three previous objectives. Broadly, my intention is to develop methods that may be of interest to other researchers studying macroevolution and macroecology. However, it is also important to ask these questions empirically, **by employing existing biological collections**. Australian reptiles are particularly well suited for these objectives because they represent a number of species rich, eco-morphologically diverse, endemic radiations. Building on phylogenomic research from my PhD and others, we now have strong species-level time-calibrated phylogenetic hypotheses for nearly all continental reptile groups (*citations*). This makes macroevolutionary inferences tractable. The Atlas of Living Australia also provides an unparalleled resource for species distributions based on museum occurrence records, for all described taxa. Given existing data and resources, and the access to material, I *honestly* believe that this is one of the best possible system to focus this research on. 

### Significance
Phylogenetic comparative methods are undergoing a period of rapid growth, largely as a result of bigger and more complete phylogenies, and the popularity of computational biology. Macroevolutionary research is also popular in science media because many of its inferences are intuitive and exciting. My proposed research builds on existing ecological theory and macroevolutionary methods, synthesizing the two. I anticipate that the output of this project will be valuable both empirically and methodologically. The construction of an ecomorphological database will provide a resource for a diverse array of biologists working from population to macroevolutionary scales. The comparative ABC method design will also make available a generalizable framework for creating evolutionary models.

## Training Objectives

## Career Development

## Sponsoring Scientist and Institution Justificiation

I have chosen Drs. Lee and O'Meara specifically for their skill sets, research interests, and proven track records of pushing the evolutionary biology community forward. They are both creative and judicious thinkers, who I admire for their contributions to science and broader education.  

**Dr. Brian O'Meara** (*Methods/Biology*) is an Associate Professor in the Department of Ecology & Evolutionary Biology at the University of Tennessee, Knoxville. 

**Dr. Michael Lee** (*Theory/Biology*) is the curator of Palaeontology at the South Australian Museum, and holds a joint appointment in the Department of Biological Sciences at Flinders University of South Australia. 

\pagebreak

## Timetable
The current timetable assumes Year 1 will be spent at the South Australian Museum in Adelaide, Australia. Years 2 and 3 will be spent at the University of Tennessee---Knoxville, USA. 
```{r echo = FALSE}
library(knitr); library(pander)

df <- data.frame(Year = c("Y1","Y2", "Y3"),
                 Research = c("Collect morphological data. Build reptile database, and integrate it into ALA[^1]",
                              "Build ABC^1^ framework for PCMs^1^. Implement empirically to reptiles",
                              "Package and release ABC framework for lineage interactions"),
                 Training = c("a", "b", "c"),
                 Dissemination = c("Release extended specimen data format for ALA, and present database",
                                  "Present methods and results at scientific conferences. Publish project   pre-prints",
                                  "Publish methods and empirical findings"))
pander(df, split.cell=20)
```

[^1]:ALA--Atlas of Living Australia; ABC--Approximate Bayesian Computation; PCM--Phylogenetic Comparative Model

## Broader Impacts Narrative






Accounting for these interactions in a phylogenetic framework is integral to understanding the evolution of both simple and complex traits, and fits with our existing knowledge of character displacement. 


# References  

# Biographic Sketch (2pg CV)
*I think I have this bit already covered*

# Current and Pending Support
The only current application for funding is through the University of California, Davis Center for Population Biology. This is a two-year self-directed fellowship. I have no other currently submitted, or planned applications for support. 

# Supplementary Documentation
*I've already submitted my workshop proposal.*

\pagebreak

## Dissertation Research Abstract (1 page)

### Macroevolution in a changing Australia

Changes to the global climate can promote macroevolutionary and macroecological turnover by either abiotic or biotic drivers, or both. Climatic changes may proceed over long or short time periods, varying in intensity from mild to extreme, and as a result, changes to macroevolutionary patterns may respond in kind. It remains unknown however, how *predictable* organismal responses may be, particularly across ecologically diverse groups. My dissertation research has largely focused on how changing environments have impacted the accumulation of diversity of Australian vertebrates since the continent's isolation ~35 million years ago. 

\  

The Miocene epoch (23–5.3 Ma) has reportedly figured prominently in the diversification of many extant faunal groups. This is likely the result of climatic instability, fluctuating atmospheric CO~2~ concentrations and floral biome turnover. Across the globe, Late Miocene cooling coincided with the birth and expansion of arid biomes and contraction of more mesic ones, as C3 forest and woodland plants were replaced by C4 savannah and grasslands. In Australia, this resulted in a dramatic restructuring of habitats, transitioning the continent from warm wet forests, to the barren red sands of today's "Outback". To investigate this idea, we can search for signature of these climatic shifts in the evolutionary history of Australia's animals. 

\  

To test for signal from empirical data, I employed phylogenetic comparative methods to link diversification patterns with climate-dictated processes. Given a set of taxa and tissue samples, we produced genetic data via traditional Sanger-sequencing and exon-capture methods. This allowed us to infer relationships among the sampled taxa, providing a phylogenetic framework. Using these phylogenies, we investigated (**1**) rates and trends in speciation and extinction, (**2**) biogeographic histories including trends in the prevailing geographic mode of speciation, and (**3**) the evolution of organismal traits. Our findings suggest that heterogeneity in the diversification of Australian animals has been largely dictated by biome restructuring through the Miocene and Plio-Pleistocene. This is evident by (**1**) increased speciation in arid and dispersal back into mesic habitats, (**2**) increased allopatry as a result of fracturing mesic habitats, and (**3**) decreasing rates of phenotypic evolution due to niche conservatism. We provide evidence that broad abiotic and biotic influences can result in similar patterns across cohabiting groups with disparate ecologies. While Australia is unique in its forms of diversity, its biogeographic and phenotypic patterns have probably been shaped by the same processes occurring elsewhere. 

\pagebreak

## Sponsoring Scientists Statement (3 pages max)

\pagebreak

## Documentation from Collection Director(s)

## Data Management Plan


# 2 Letters of Reference (using Fastlane)